### PR TITLE
Meenunat tl gt pass

### PR DESCRIPTION
**Pull request by meenunat**
Currently using the simulator/camera image - with ground truth stop line information from stop line config - the next closest way point to the light is predicted. I am printing light state and next closest waypoint at a point- this log infor is p=for debug purpose we can comment it out later.

### CODE REVIEW

File: ros/src/tl_detector/tl_detector.py [hunk 1]
```diff
@@ -100,8 +139,32 @@ def get_closest_waypoint(self, pose):
             int: index of the closest waypoint in self.waypoints
 
         """
-        #TODO implement
-        return 0
+        wp_id = None
+        wps = waypoints.waypoints
+
+        x = pos_x
+        y = pos_y
```
Comment (edufford) [inline ros/src/tl_detector/tl_detector.py]: Do these need to be copied?

File: ros/src/tl_detector/tl_detector.py [hunk 2]
```diff
@@ -10,18 +10,33 @@
...
+GROUND_TRUTH_PASS = True  
+#Variable to use ground truth data until the classifier has been developed
 
 class TLDetector(object):
     def __init__(self):
         rospy.init_node('tl_detector')
 
+		#Private Parameter to differentiate simulation or Site Launch  
+		#Default - site launch
+        self.isSimulation = bool(rospy.get_param('~isSimulation', False))
```
Comment (edufford) [inline ros/src/tl_detector/tl_detector.py]: Is this flag used somewhere?

File: ros/src/tl_detector/tl_detector.py [hunk 3]
```diff
@@ -134,9 +197,9 @@ def process_traffic_lights(self):
         light = None
 
         # List of positions that correspond to the line to stop in front of for a given intersection
-        stop_line_positions = self.config['stop_line_positions']
+        #stop_line_positions = self.config['stop_line_positions']
         if(self.pose):
-            car_position = self.get_closest_waypoint(self.pose.pose)
+        	car_position = self.get_closest_waypoint(self.pose.pose)
```
Comment (edufford) [inline ros/src/tl_detector/tl_detector.py]: Although this isn't used now, may want to update the function arguments to match the definition.

File: ros/src/tl_detector/tl_detector.py (hunk 2 above)
Comment (meenunat) [reply ros/src/tl_detector/tl_detector.py]: I updated the algorithm after debug- but forgot to clean this up. Will update the branch with this fixed.

File: ros/src/tl_detector/tl_detector.py (hunk 1 above)
Comment (meenunat) [reply ros/src/tl_detector/tl_detector.py]: I updated the algorithm after debug- but forgot to clean this up. Will update the branch with this fixed.

File: ros/src/tl_detector/tl_detector.py (hunk 3 above)
Comment (meenunat) [reply ros/src/tl_detector/tl_detector.py]: Yep - will fix this.

**Comment by meenunat** [comment]
Yes -will update all the mentioned changes and re-check in.

**Comment by meenunat** [comment]
@edufford @shrikondra - Can you review the latest changes and approve the merge?

**Pull request merged**